app/views/catalog/_header_icons.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= geoblacklight_icon(document[Settings.FIELDS.GEOM_TYPE]) %>
-<%= geoblacklight_icon(document[Settings.FIELDS.PROVENANCE]) %>
-<%= geoblacklight_icon(document[Settings.FIELDS.RIGHTS]) %>
\ No newline at end of file
+<%= geoblacklight_icon(document[Settings.FIELDS.GEOM_TYPE], classes: 'svg_tooltip') %>
```

Comment:
Is this being overwritten by https://github.com/geoblacklight/geoblacklight/pull/934/files#diff-6ca53a94b233e28fba3b527f2fe59f72R331 ?

Reply:
https://github.com/geoblacklight/geoblacklight/pull/934/files#diff-6ca53a94b233e28fba3b527f2fe59f72R331 is in the `relations_icon` helper method.  Both are calling `geoblacklight_icon` and sending in `classes`.

Reply:
ah.. of course :)